### DISCUSSION TITLE
Question about UK Biobank T1_brain_to_MNI.nii.gz

### DISCUSSION THREAD

**Discussion by jiashizuo** [McIntosh-Lab/tvb-ukbb#194] [Help] [answered]
Hello, I found an issue of temporal lobe distortion during the non-linear registration process of T1 images while reading "Image Processing and Quality Control for the first 10,000 Brain Imaging Datasets from UK Biobank". I would like to ask if the T1_brain_to_MNI.nii.gz file provided by UK Biobank has any temporal lobe distortion issue?
![Snipaste_2024-04-18_21-54-02](https://github.com/McIntosh-Lab/tvb-ukbb/assets/68786052/145dab87-4a46-4085-9095-263eec9ec736)

**Comment by shen4brains** [comment, marked as answer]
Hello! This repository is for The Virtual Brain's implementation of the UK Biobank pipeline. We have not yet tested our implementation on the UK Biobank data, and instead have generalized the pipeline for other datasets that span the lifecourse. For questions related to UK Biobank data, I suggest you get in touch with the UK Biobank group directly.

**Reply by jiashizuo** [reply]
Thank you for your reply！
